docker/compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.4'
```

Comment:
if the compose file is in a folder named "docker", the default ["project name"](https://docs.docker.com/compose/reference/#use--p-to-specify-a-project-name), which is the prefix to the names of all the things it creates, will be "docker." probably better to put this file at the root of the repo.

Reply:
I have these files in /docker to take advantage of the fact that GitHub allows for multiple README files in different folders and the current instructions are to move the file out anyways, but I could refactor the current docker README into a wiki page if that would work better.

Reply:
that's a good reason.

some thoughts: you don't _have_ to put everything in README.md files - people can click on .md files with a different name and github will format it nicely, just not on the same page as the folder. you can also include links from one .md file to another in their text, including from one readme to some other more specific file. this is a little more discoverable than the wiki, though wikis are fine too.

i'll leave it to you to decide whether to do any of that or keep it as it is.

Reply:
File paths refactored; README.md has been converted to a wiki page